packages/addresses/src/__tests__/program-derived-address-test.ts

```diff
@@ -0,0 +1,141 @@
+import { Base58EncodedAddress } from '../base58';
+import { getProgramDerivedAddress } from '../program-derived-address';
+
+describe('getProgramDerivedAddress()', () => {
+    it('fatals when supplied more than 16 seeds', async () => {
```

Comment:
huh interesting was this always a property of pda generation? i never had cause to check, and more than 16 is certainly pointless, but i always assumed it was happy hashing more inputs in forever